OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyOptionsLogic.cs

```diff
@@ -101,18 +101,25 @@ void RebuildOptions()
 				}
 
 				var checkbox = checkboxColumns.Dequeue();
-				var optionValue = new CachedTransform<Session.Global, Session.LobbyOptionState>(
-					gs => gs.LobbyOptions[option.Id]);
+				var optionEnabled = new PredictedCachedTransform<Session.Global, bool>(
+					gs => gs.LobbyOptions[option.Id].IsEnabled);
+
+				var optionLocked = new CachedTransform<Session.Global, bool>(
+					gs => gs.LobbyOptions[option.Id].IsLocked);
```

Comment:
All the expensive operations are done during the update, so this doesn't appear to need a cached transform.

Reply:
Actually nvm, I read the func wrongly, we're comparing gs.